Agentic AI/MCP/fromAi adaptions (#6978)
camunda/camunda-docs: docs/components/modeler/feel/builtin-functions/feel-built-in-functions-miscellaneous.md

Returns the unmodified `value` parameter.

> In addition to the previous overload, it also accepts an optional `description` parameter to provide a textual description of the value. The description needs to be `null` or a string constant.

Reviewer: ```suggestion
In addition to the previous overload, it also accepts an optional `description` parameter to provide a textual description of the value. The description must be `null` or a string constant.
```

In addition to the previous overload, it also accepts an optional `options` parameter to provide additional options for the integration handling the value definition.

> - The options parameter needs to be `null` or a context (map) containing only constant values. For example, function calls within options are not supported.

Reviewer: ```suggestion
- The options parameter must be `null` or a context (map) containing only constant values. For example, function calls within options are not supported.
```
